usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -8129,6 +8129,13 @@ CK_RV ep11tok_generate_key_pair(STDLL_TokData_t * tokdata, SESSION * sess,
         goto error;
     }
 
+    rc = update_ep11_attrs_from_blob(tokdata, sess, public_key_obj, FALSE);
+    if (rc != CKR_OK) {
+        TRACE_ERROR("%s update_ep11_attrs_from_blob failed with rc=0x%lx\n",
+                    __func__, rc);
+        goto error;
+    }
+
     /* Copy CKA_MODULUS and CKA_PUBLIC_EXPONENT attributes from
```

Comment:
ok, yes, that was missing. Not all attrs handled by update_ep11_attrs_from_blob() may be applicable for public keys.

Reply:
... but this is handled by checking each attr value for CK_UNAVAILABLE_INFORMATION. OK.